cfy_cluster_manager/main.py

```diff
@@ -43,6 +41,9 @@
 CLUSTER_CONFIG_FILE_NAME = 'cfy_cluster_config.yaml'
 CLUSTER_INSTALL_CONFIG_PATH = join(os.getcwd(), CLUSTER_CONFIG_FILE_NAME)
 
+SYSTEMD_RUN_UNIT_NAME = 'cfy_cluster_manager_{}'
+INITIAL_INSTALL_DIR = join('/etc/cloudify', '.installed')
```

Comment:
Probably not a good idea to use join on this- this script might be runnable on windows at some point, and you already joined the previous parts by hand.

Reply:
Changed to '/etc/cloudify/.installed'